src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -4487,7 +4495,7 @@ components:
         checksums:
           type: array
           description: >
-            Always present for files - checksums; always includes crc32c. May include md5.
+            Always present for files - checksums; May include md5 or crc32c.
```

Comment:
Thanks for updating this!  Can I suggest `May include md5 and/or crc32c` instead?

Reply:
ah yep